service/static/index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Wishlist RESTful Service</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script type="text/javascript" src = "static/js/jquery-3.1.1.min.js"></script>
+    <script type="text/javascript" src="static/js/rest_api.js"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+  </head>
+  <body>
+    <div class="container">
+        <div class="page-header">
+            <h1>Wishlist REST API Service</h1>
+        </div>
+
+        <!-- Flash Message -->
+        <div class="table-responsive">
+          <table class="table">
+            <tr><td>Status:</td><td><span id="flash_message"></span></td></tr>
+          </table>
+        </div>
+        <ul class="nav nav-tabs">
+          <li class="active"><a data-toggle="tab" href="#wishlist">Wishlist</a></li>
```

Comment:
Can we add the ids here? So that we can specify to selenium where to click to change the tabs.

Reply:
Oh you mean to add an id for ul?

Reply:
yeah. We need the automation to find these items so it can click and change the values. It can only change the values of the items it sees on the page. Basically selenium acts like a user using the website.